--- src/gmixer.old	2010-08-31 10:54:43.000000000 +0200
+++ src/gmixer	2010-08-31 10:55:07.747000063 +0200
@@ -36,7 +36,6 @@
 import gtk
 import gtk.glade
 
-from gtktrayicon import TrayIcon
 import gst
 import gst.interfaces
 if gst.pygst_version < (0, 10, 8):
@@ -803,18 +802,11 @@
                     break;
 
     def _init_tray_icon(self):
-        self._tray_image = gtk.Image()
-        self._tray_image.set_property("has-tooltip",True)
-
-        self._eventbox = gtk.EventBox()
-        self._eventbox.connect("button_press_event", self._on_tray_press)
-        self._eventbox.connect("scroll-event",self._on_tray_scroll)
-        self._eventbox.add(self._tray_image)
-
-        self._tray = TrayIcon("GMixer")
-        self._tray.add(self._eventbox)
+        self._tray = gtk.StatusIcon()
+        self._tray.connect("button_press_event", self._on_tray_press)
+        self._tray.connect("scroll-event",self._on_tray_scroll)
         self._update_tray()
-        self._tray.show_all()
+        self._tray.set_visible(True)
 
     def _get_master_track(self):
         for track in self._current_mixer.list_tracks():
@@ -828,8 +828,8 @@ class GMixer(gobject.GObject):
         elif frac > 0.33: img = "audio-volume-medium"
         elif frac > 0.0: img = "audio-volume-low"
         else: img = "audio-volume-muted"
-        self._tray_image.set_from_icon_name(img, gtk.ICON_SIZE_SMALL_TOOLBAR)
-        self._tray_image.set_tooltip_text(_("Volume : %d%%"%(frac*100)))
+        self._tray.set_from_icon_name(img)
+        self._tray.set_tooltip(_("Volume : %d%%"%(frac*100)))
 
     def program_is_alive(self, pid, cmd):
         try:
